resources/seccomp/x86_64-unknown-linux-musl.json

```diff
@@ -1254,6 +1254,18 @@
                     }
                 ]
             },
+            {
+                "syscall": "ioctl",
+                "args": [
```

Comment:
we can actually add a comment here on why we need it.

Reply:
Extended the comment with the use-case.